index.html

```diff
@@ -1,36 +1,235 @@
+  <!--WORK 3-->
+  <div class="card">
+    <img class="card-picture" src="/images/picture3.PNG" alt="work example" />  
+    <div class="card-content">
+    <h2 class="title"> Tonic</h2>
+      <div class="info">
+        <span>CANOPY</span>
+        <span class="punto"></span>
+        <span>Back End Dev</span>
+        <span class="punto"></span>
+        <span>2015</span>
+      </div>
+      <p class="description"> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+      <div class="pr-languages">
+        <div class="pr-lang"><span>html</span></div>
+        <div class="pr-lang"><span>css</span></div>
+        <div class="pr-lang"><span>javaScript</span></div>
+      </div>
+      </div>
+      <div class="project-button">
+        <button class="btn" type="button">See Project</button>
+      </div>
+    </div>
+  <!--END OF WORK 3-->
+  <!--WORK 4-->
+  <div class="card">
+    <img class="card-picture" src="/images/picture4.PNG" alt="work example" />  
+    <div class="card-content">
+    <h2 class="title">Multi-Post Stories</h2>
+      <div class="info">
+        <span>CANOPY</span>
+        <span class="punto"></span>
+        <span>Back End Dev</span>
+        <span class="punto"></span>
+        <span>2015</span>
+      </div>
+      <p class="description"> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+      <div class="pr-languages">
+        <div class="pr-lang"><span>html</span></div>
+        <div class="pr-lang"><span>css</span></div>
+        <div class="pr-lang"><span>javaScript</span></div>
+      </div>
+      </div>
+      <div class="project-button">
+        <button class="btn" type="button">See Project</button>
+      </div>
+    </div>
+  </div>
+<!--END OF WORK 4-->
+<!--END OF WORK SECTIONS-->
+<!--ABOUT ME SECTION-->
+<section id="about-me">
+    <div class="contenedor">
+      <div class="about-me-content">
+      <h3 class="title"> About Myself</h3>
+      <p class="description"> I'm a software developer! I can help you build a product ,feature or website. Look through some of my work and experience! if you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      <p class="connect">LETS CONNECT</p>
+          <div class="social-network">
+              <ul>
+                <li><img src="images/twitter-logo.svg" alt="twitter" /></li> 
+                <li><img src="images/Linkedin.svg" alt="linkedin"/></li>
+                <li><img src="images/github.svg" alt="github"/></li>
+                <li><img src="images/Vector2.svg" alt="#"/></li>
+                <li><img src="images/Vector.svg" alt="#"/></li>
+             </ul>
+          </div>
+          <div class="project-button">
+            <button class="btn" type="button">Get my resume</button>
+          </div>
+        </div>
+      </div>
+  </section>
+<!--BEGGINING OF LANGUAGES-->
+<!--codigo de w3school-->
+<div class="languages">
```

Comment:
- [ ] [OPTIONAL] Greate work implementing the changes 👍🏼 however you may consider removing the little margin on to of the Language so your design could be identical to Figma design, and also please consider adding some margin to the languages cards ```JavaScript```, ```HTML```, ```CSS```

|Your design|Figma design|
|-|-|
|![Screenshot_20220528_173001](https://user-images.githubusercontent.com/25227022/170834312-9a0034f0-d139-437b-b720-f41c7e860492.png)|![Screenshot_20220528_173021](https://user-images.githubusercontent.com/25227022/170834317-b6e0a27f-b0c1-4a9a-86ef-0307e44c6139.png)|

Reply:
I'll apply those changes. Thanks for the feedback @ismailco :smile: